api/models/Tag.js

```diff
@@ -279,7 +279,7 @@ module.exports = bookshelf.Model.extend({
   },
 
   nonexistent: (names, communityIds) => {
-    if (names.length === 0 || communityIds.length === 0) return Promise.resolve({})
+    if (names.length === 0 || !communityIds || communityIds.length === 0) return Promise.resolve({})
```

Comment:
hm, sorry again. I COULD switch it but I think it would make it less clear. So I'm still going to leave it